twake/frontend/src/app/scenes/Apps/Messages/Input/Input.tsx

```diff
@@ -139,6 +145,7 @@ export default (props: Props) => {
       {!hasEphemeralMessage && !props.messageId && (
         <InputOptions
           inputValue={content}
```

Comment:
It seems like you did not finish this one, create a "inputEmpty={!!content || hasFilesAttached}" props (hasFilesAttached is to define of course)

Reply:
Done :+1: